$NetBSD: patch-rebar.config,v 1.2 2021/05/20 20:40:48 triaxx Exp $

* On NetBSD and OpenBSD there is no libdl, as the required functionality is
  already built into libc.
* Use pkgsrc dependencies.

--- rebar.config.orig	2021-03-24 17:32:32.000000000 +0000
+++ rebar.config
@@ -26,13 +26,11 @@
 
 {port_env, [{"ERL_LDFLAGS", " -L$ERL_EI_LIBDIR -lei"},
             {"CFLAGS", "$CFLAGS"},
-            {"^((?!openbsd).)*$", "LDFLAGS", "$LDFLAGS -ldl"},
+            {"^((?!(net|open)bsd).)*$", "LDFLAGS", "$LDFLAGS -ldl"},
             {"LDFLAGS", "$LDFLAGS"}]}.
 
 {port_specs, [{"priv/bin/eimp", ["c_src/eimp.c"]}]}.
 
-{deps, [{p1_utils, ".*", {git, "https://github.com/processone/p1_utils", {tag, "1.0.22"}}}]}.
-
 {clean_files, ["c_src/eimp.gcda", "c_src/eimp.gcno"]}.
 
 {cover_enabled, true}.
